tests/polynomial/elementwise_to_affine.mlir

```diff
@@ -1,5 +1,6 @@
 // RUN: heir-opt --convert-elementwise-to-affine %s | FileCheck --enable-var-scope %s
-
+// RUN: heir-opt --convert-elementwise-to-affine=ops-to-consider="polynomial.add,polynomial.mul_scalar" %s | FileCheck --enable-var-scope %s
```

Comment:
for simplicity, should we rename to `restrict-ops`, and `restrict-dialects`? welcome for any feedback!

Reply:
these names sounds good.